README.md

```diff
@@ -1,20 +1,78 @@
 # Create Custom SCW
 
-### Intall Dependency
 
+
+### How to set up the bundler?
+
+- First clone the repo `https://github.com/eth-infinitism/bundler.git`.
+
+- Now install all the modules in the root directory `budler/` by doing:
+```shell 
+yarn
+```
+
+- Then we need to build individual packages and resolve dependancy issues. For this first create a file `testnet-mnemonic.txt` and paste your private key inside it. Now in the terminal:
+```shell
+MNEMONIC_FILE=testnet-mnemonic.txt yarn preprocess
+```
+
+- Now replace the mneumonic present at `packages/bundler/localconfig/mnemonic.txt` with your private key.
+
+- We also need to configure the `bundler.config.json` at `packages/bundler/localconfig/bundler.config.json` with the following changes:
+```shell
+{
+  "mnemonic": "./localconfig/mnemonic.txt",
+  "network": "goerli",
+  "beneficiary": "<YOUR_PUBLIC_KEY>",
+  "port": "3000",
+  "helper": "0x214fadBD244c07ECb9DCe782270d3b673cAD0f9c",
+  "entryPoint": "<YOUR_ENTRY_POINT_ADDRESS>",
+  "minBalance": "0",
+  "gasFactor": "1"
+}
+```
+
+-  As we have changed the mneumonic to private key, we need to make some changes to incoporate this:
+    - We need to make changes in the `hardhat.config.ts` present inside `/bundler` package. Replace the `accounts` in `getNetwork` with:
+    ```shell
+        accounts: [mnemonic]
+    ```
+    - In the `packages/bundler/src/runBundler.ts` file we need to change two things. First, we should change the provider by giving a ethers JsonRpcProvider as follows:
+        ```shell
+        const provider: BaseProvider =
+            // eslint-disable-next-line
+            config.network === 'hardhat' ? require('hardhat').ethers.provider :
+            new ethers.providers.JsonRpcProvider(`https://NETWORK.infura.io/v3/YOUR_INFURA_ID`)
```

Comment:
I don't think we need this, I think we were failing mostly due to gas.

Reply:
I tried removing this, but the txn is failing